tests/yam/agama/agama_cli.pm

```diff
@@ -0,0 +1,69 @@
+## Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Perform a default installation using only CLI while exercising various commands.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use testapi;
+use power_action_utils qw(power_action);
+use Utils::Architectures qw(is_s390x);
+use Utils::Backends qw(is_svirt);
+
+sub agama_config_edit {
+    my $regex = shift;
+
+    script_run('agama config edit', timeout => 0);
+    wait_still_screen();
+    type_string($regex);
+    type_string(":wq\n");
+    wait_still_screen();
+}
+
+sub run {
+    my $self = shift;
+
+    validate_script_output('agama 2>&1 | tee', qr/Agama's command-line interface/, proceed_on_failure => 1);
```

Comment:
removing all the error messages was not part of the suggestion, those actually help.